composer.json

```diff
@@ -11,7 +11,7 @@
     ],
     "require": {
         "php": "^7.1|^8",
-        "tarantool/queue": "^0.9"
+        "tarantool/queue": "^0.10"
```

Comment:
Hi @luscinium, thanks for the PR. Unfortunately, it is not quite complete. In `tarantool/queue` v0.10, support for PHP 7.1 has been removed, so the PHP version requirements and the `.github/workflows/qa.yml` file must be updated accordingly.  Is this something you're willing to work on?

Reply:
Sure. I've updated the QA-workflow to remove 7.1 and add 8.1. All tests succeeded.

Reply:
Great! There is one minor thing left, the PHP requirement in composer.json needs to be updated to `"php": "^7.2.5|^8"`.